build.sbt

```diff
@@ -31,6 +31,11 @@ lazy val restaurants = (project in file("restaurants"))
   .settings(commonSettings)
   .settings(Seq(name := "restaurants"))
 
+lazy val games = (project in file("games"))
+  .dependsOn(common)
+  .settings(commonSettings)
+  .settings(Seq(name := "games"))
```

Comment:
Note, you shouldn't need to do this as it will use the name of the val I believe unless you are doing it for some other reason?